libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -295,6 +295,9 @@ static const AP_RCProtocol::SerialConfig serial_configs[] {
 #endif
     // CrossFire:
     { 416666,  0,   1, false },
+    // CRSFv3 can negotiate higher rates which are sticky on soft reboot
```

Comment:
should check AP_RCPROTOCOL_ELRS_ENABLED?

Reply:
Fixed